config.c

```diff
@@ -53,6 +53,10 @@
 #include <time.h>
 #endif
 
+#ifdef __OpenBSD__
+#include <time.h>
```

Comment:
Without this include on OpenBSD you're accessing an opaque struct when setting seconds to 0.